src/main/java/Calculator/OperationStrategy.java

```diff
@@ -0,0 +1,5 @@
+package Calculator;
+
+public interface OperationStrategy {
```

Comment:
```suggestion
@FunctionalInterface
public interface OperationStrategy {
```

꼭 필요하지 않지만 이 인터페이스의 쓰임새를 조금 더 나타내려면 명시해주시는게 좋다고 생각합니다.
그리고 별도의 인터페이스 없이 `BiFunction<Integer, Integer, Integer>` 을 사용하시는 방법도 있습니다.